base/show.jl

```diff
@@ -554,7 +555,7 @@ function make_typealias(@nospecialize(x::Type))
         for name in names(mod)
             if isdefined(mod, name) && !isdeprecated(mod, name) && isconst(mod, name)
                 alias = getfield(mod, name)
-                if alias isa Type && !has_free_typevars(alias) && !isvarargtype(alias) && !print_without_params(alias) && x <: alias
```

Comment:
Should this change be removed for the backport to 1.6?

Reply:
Yes, this commit can be ignored for the backport